server/base/src/main/java/org/apache/accumulo/server/security/AuditedSecurityOperation.java

```diff
@@ -685,8 +685,8 @@ public boolean hasSystemPermission(TCredentials credentials, String user,
       "action: %s; targetTable: %s:%s";
 
   @Override
-  public boolean canOnlineOfflineTable(TCredentials credentials, TableId tableId, FateOperation op,
-      NamespaceId namespaceId) throws ThriftSecurityException {
+  public boolean canOnlineOfflineOnDemandTable(TCredentials credentials, TableId tableId,
```

Comment:
Could these methods be combined into something like `canChangeTableState(...)` for the method name?  The permission could be a general allowed to change state and not for a specific state.

Reply:
Renamed to `canChangeTableState`